.github/workflows/publish.yml

```diff
@@ -12,6 +12,13 @@ on:
       - published
       - edited
 
+# We set `concurrency` to prevent having this workflow being run on code that is not up-to-date on a PR (a user make multiple push in a quick manner).
```

Comment:
```suggestion
# Prevent concurrent run of `publish` workflow
# We  set `concurrency` to prevent having this workflow being run on code that is not up-to-date on a PR (a user make multiple push in a quick manner).
```

Reply:
I've simplified the comment